modules/nuxt-zod.yml

```
@@ -0,0 +1,16 @@
+name: nuxt-zod
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)